config/methods.php

```diff
@@ -1,635 +0,0 @@
-<?php
-
-use Kirby\Cms\App;
-use Kirby\Cms\Blocks;
-use Kirby\Cms\Collection;
-use Kirby\Cms\File;
-use Kirby\Cms\Files;
-use Kirby\Cms\Html;
-use Kirby\Cms\Layouts;
-use Kirby\Cms\Page;
-use Kirby\Cms\Pages;
-use Kirby\Cms\Structure;
-use Kirby\Cms\Url;
-use Kirby\Cms\User;
-use Kirby\Cms\Users;
-use Kirby\Content\Content;
-use Kirby\Content\Field;
-use Kirby\Data\Data;
-use Kirby\Exception\Exception;
-use Kirby\Exception\InvalidArgumentException;
-use Kirby\Exception\NotFoundException;
-use Kirby\Image\QrCode;
-use Kirby\Toolkit\A;
-use Kirby\Toolkit\Dom;
-use Kirby\Toolkit\Str;
-use Kirby\Toolkit\V;
-use Kirby\Toolkit\Xml;
-use Kirby\Uuid\Uuid;
-
-/**
- * Field method setup
- */
-return function (App $app) {
-	return [
-
-		// states
-
-		/**
-		 * Converts the field value into a proper boolean and inverts it
-		 */
-		'isFalse' => function (Field $field): bool {
-			return $field->toBool() === false;
-		},
```

Comment:
`Comment:` Alternatively to deleting the file we could add some legacy code to ease breaking changes where users require the config file to extend certain field methods. But not sure if this really helps or just delays the pain.

```php
'isFalse' => fn (Field $field) => $field->isFalse()
```